ftl/common/builder.py

```diff
@@ -114,7 +116,10 @@ def Build(self):
     def AppendLayersIntoImage(self, lyr_imgs):
         for i, lyr_img in enumerate(lyr_imgs):
             if i == 0:
-                result_image = lyr_img
+                if isinstance(lyr_img, layer_builder.PackageLayerBuilder):
```

Comment:
This isn't a great pattern. Is there some way to check by calling a method on the object itself?

Reply:
done, using hasattr